website/blog/2020-03-24-version-0.62.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Announcing React Native 0.62 with Flipper
+author: Rick Hanlon
+authorTitle: React Native Core at Facebook
+authorURL: https://twitter.com/rickhanlonii
+authorImageURL: https://avatars3.githubusercontent.com/u/2440089?s=460&v=4
+authorTwitter: rickhanlonii
+category: announcements
+---
+
+Today we’re releasing React Native version 0.62 which includes support for Flipper by default.
+
+This release comes in the midst of a global pandemic. We’re releasing this version today to respect the work of hundreds of contributors who made this release possible and to prevent the release from falling too far behind master. Please be mindful of the reduced capacity of contributors to help with issues and prepare to delay upgrading if necessary.
+
+## Flipper by default
+
+[Flipper](https://fbflipper.com/) is a developer tool for debugging mobile apps. It’s popular in the Android and iOS communities, and in this release we’ve enabled support by default for new and existing React Native apps.
+
+Flipper provides the following features out of the box:
+
+- Network Inspector
+- Logout Inspector
+- Crash Reporter
+- Device logs
+
+There is also a marketplace of custom Flipper plugins you can install:
+
+[TODO: Flipper Image]
+
+For more information, check out the Flipper documentation here.
+
+## Better dark mode support
+
+We’ve added a new Appearance module to provide access to the user's appearance preferences, such as their preferred color scheme (light or dark).
+
+```
+const colorScheme = Appearance.getColorScheme();
+if (colorScheme === 'dark') {
+  // Use dark color scheme
+}
+```
+
+We’ve also added a hook to subscribe to state updates to the users preferences automatically:
+
+```
+import {Text, useColorScheme} from 'react-native';
+
+const MyComponent = () => {
+  const colorScheme = useColorScheme();
+  return <Text>useColorScheme(): {colorScheme}</Text>;
+};
+```
+
+See the docs for [Appearance](https://reactnative.dev/docs/appearance) and [useColorScheme](https://reactnative.dev/docs/usecolorscheme) for more information.
+
+## Moving AppleTV to react-native-tvos
+
+As part of our Lean Core effort and to bring AppleTV in line with other platforms like React Native Windows and React Native MacOS, we’ve started to remove AppleTV specific code from core.
```

Comment:
```suggestion
As part of our [Lean Core effort](/LINK) and to bring AppleTV in line with other platforms like React Native Windows and React Native MacOS, we’ve started to remove AppleTV specific code from core.
```
nit: maybe worth adding the link to the lean core issue? (is that still open? it's been quite a long time since last I've checked)

Reply:
Looks like it's still open https://github.com/facebook/react-native/issues/23313

```suggestion
As part of our [Lean Core effort](https://github.com/facebook/react-native/issues/23313) and to bring AppleTV in line with other platforms like React Native Windows and React Native MacOS, we’ve started to remove AppleTV specific code from core.
```

Reply:
There’s also https://reactnative.dev/blog/#lean-core

Reply:
```suggestion
As part of our [Lean Core effort](https://reactnative.dev/blog/#lean-core) and to bring Apple TV in line with other platforms like React Native Windows and React Native MacOS, we’ve started to remove Apple TV specific code from core.
```